src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -294,6 +296,11 @@ public void timeStep(Time t, DailyStats dStats) {
         for (int i = 0; i < nDays; i++) {
             DailyStats dStats = new DailyStats(t);
             implementLockdown(t);
+
+            if (t.getAbsDay() <= externalInfectionDays) {
```

Comment:
Should this be `<`, rather than `<=`?

Reply:
Unsure since we don't infect on day 0, <= gives us the full, e.g. 55 days, of infections. While < only gives 54 days worth of infections.

I'm not really sure what's best. I guess if we are, for example, starting a lockdown on the same day we might want it to be < so that on day 55 we get the lockdown and no more infections.

@paulbessell  what do you prefer?

Reply:
I suggest leaving it as is, but adding a comment giving your explanation.